src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -1556,6 +1558,10 @@ TieSegment* SlurTieLayout::tieLayoutBack(Tie* item, System* system, LayoutContex
     segment->setSystem(system);
     segment->resetAdjustmentOffset();
 
+    if (chord) {
+        segment->setStaffIdx(chord->vStaffIdx());
```

Comment:
This feels slightly hacky (and slightly dangerous) cause you are changing the staffIdx of only one of the TieSegments, so you end up with mismatching tracks between the Tie item and its two TieSegments. Anyway, this is such an edge case that, if it works properly, I think we can keep the hack, for once, as it does make it all quite simple. Maybe do write a small hack-warning comment to explain that though :)

Reply:
Ah yeah, having a `vStaffIdx` field for tie segments would absolutely be safer - I can try to make that work